float64[2]    robot_pos